### ISSUE TITLE
Unable to connect to AWS MSK Kafka via TLS

### ISSUE THREAD

**Issue by den-is** [fluent/fluent-bit#1495]
## Bug Report

Unable to send messages to AWS MSK Managed Kafka via secure TLS connection.
Pipeline looks like this:
`node-with-agent > Kafka > Fluentd > ElasticSearch`

**Environment:**
Fluentbit td-agent-bit v1.2.2
CentOS7 EC2
AWS MSK using Apache Kafka v2.1.0
Tail - input
Using one filter to add field

**Logs**
Debug logging adds nothing informative or interesting in addition to only below error records.
```
[2019/08/05 11:56:34] [ info]  collectors     :
[2019/08/05 11:56:34] [error] [out_kafka] cannot configure 'security.protocol' property
[2019/08/05 11:56:36] [error] [out_kafka] fluent-bit#producer-1: [thrd:b-2.kafkanode.sqjyzw.c2.kafka.us-east-1.amazonaws.com:9094/]: 3/3 brokers are down
...
```
**Config**
```
[OUTPUT]
    Name kafka
    Match *
    Brokers kafka1:9094,kafka2:9094,kafka3:9094    # shortened actual worker names
    Topics fluentd
    Timestamp_Key  @timestamp
    Timestamp_Format iso8601
    rdkafka.security.protocol ssl
    rdkafka.log.connection.close false
    rdkafka.request.required.acks 1
    rdkafka.queue.buffering.max.kbytes 10240
```

**Additional info**
Absolutely same pipeline, fluent-bit version and config (but with more filters) is working in the same us-east-1 AWS in kubernetes cluster cluster as daemon set which is sending data into the same AWS MSK.

Might be related to https://github.com/fluent/fluent-bit/issues/632

**UPD:** AWS MSK upgrade to latest kafka v2.2.1 didn't help either

**Comment by edsiper** [comment]
looks like you are using our Centos Package, sadly that version is not building librdkafka with SSL/TLS support, that's getting fixed in the upcoming v1.3 release.

**Comment by edsiper** [comment]
v1.3.1 is out.
